File: c#/UnitTestsIntro/TimeUtility.cs
namespace UnitTestsIntro;

public class TimeUtility
{
    public string GetTimeOfDay()
    {
        var time = DateTime.Now;
        
        return time.Hour switch
        {
            >= 0 and < 6 => "Night",
            >= 6 and < 12 => "Morning",
            >= 12 and < 18 => "Afternoon",
            _ => "Evening"
        };
    }
}

File: solutions/java/src/main/java/unit/tests/intro/TimeUtility.java
package unit.tests.intro;

import java.time.LocalTime;

public class TimeUtility {
    public String getTimeOfDay() {
        var time = LocalTime.now();

        if (time.getHour() < 6) {
            return "Night";
        } else if (time.getHour() < 12) {
            return "Morning";
        } else if (time.getHour() < 18) {
            return "Afternoon";
        }
        return "Night";
    }
}